.github/dependabot.yml

```diff
@@ -39,6 +39,7 @@ updates:
           - tauri-bundler
           - tauri-macos-sign
           - tauri-driver
+          - "@tauri-apps/*"
```

Comment:
Those aren't Rust dependencies though right? I think we need a separate group within the npm package-ecosystem here.

Reply:
D'oh! That's what I get for opening my editor before coffee.

Reply:
fixed